aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -151,6 +139,10 @@ The only modules required to run an ASP.NET Core app are the Anonymous Authentic
 
 ![IIS Manager open to Modules with the minimum module configuration shown](modules/_static/modules.png)
 
+The URI Caching Module (`UriCacheModule`) allows IIS to cache website configuration at the URL level. Without this module, IIS must read and parse configuration on every request, even when the same URL is repeatedly requested. Parsing the configuration every request results in a significant performance penalty. *Although the URI Caching Module isn't strictly required for a hosted ASP.NET Core app to run, we recommend that the URI Caching Module be enabled for all ASP.NET Core deployments.*
```

Comment:
Many thanks for your update. The additional notes here looks great and accurate to me.